### ISSUE TITLE
audit-avt1-guidelines-color: inline frame title

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1493]
## Environment:
> Operating system: Mac
> Browser: Chrome
> AVT :one:
> Page: [Guidelines - Color: Overview](https://www.carbondesignsystem.com/guidelines/color/overview)

## Detailed Description:
Inline frame does not have a 'title' attribute

<img width="1680" alt="Screen Shot 2020-08-04 at 11 27 48 AM" src="https://user-images.githubusercontent.com/32720851/89313030-f9304880-d645-11ea-97b5-cfb7be7a4f00.png">

## What to do:
Use the "title" attribute of the <iframe> element to provide a unique, non-empty title that describes its contents. For example:

## Why is this important?
An inline frame’s title provides a label for the frame that describes its content to the user. With this information a user can determine which frame to enter and explore in detail.
